docs/README.md

```diff
@@ -2,12 +2,16 @@
 
 We currently have the following components available for Angular applications:
 
+-   [App Bar](https://github.com/brightlayer-ui/angular-component-library/blob/dev/docs/AppBar.md)
```

Comment:
might want to point everything to the master branch, or use relative links instead

Reply:
Just pushed out a fix, it's using 'master' now.  Good catch!